devtools/devframe/client/src/panel-styles.css

```
@@ -0,0 +1,6 @@
+/* The devtools sources live outside this package's vite root, so tailwind's
+   automatic source detection misses them — scan them explicitly. The shared
+   sources contain the dynamic JSON value color classes. */
+@source '../../../src/panel';
+@source '../../../src/shared';
+@import '../../../src/panel/styles.css';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)